sdk/Lusid.Sdk.Tests/tutorials/Instruments/CreditDefaultSwapExamples.cs

```diff
@@ -175,5 +176,90 @@ public void CreditDefaultSwapGetPortfolioCashFlowsExample(ModelSelection.ModelEn
+                {new VendorModelRule(VendorModelRule.SupplierEnum.IsdaCds, "VendorDefault", "CreditDefaultSwap")};
+            var pricingContext = new PricingContext(options: pricingOptions, modelRules: modelRules);
+            // tell Lusid where to find data for IsdaCreditCurves; the IsdaYieldCurve is found according to a default rule
+            var mktRules = new List<MarketDataKeyRule> {new MarketDataKeyRule("Credit.IsdaCreditCurve.*.*", "Lusid", testScope, MarketDataKeyRule.QuoteTypeEnum.Rate, "mid")};
+            var mktOptions = new MarketOptions("Lusid", null, testScope);
+            var mktContext = new MarketContext(mktRules, options: mktOptions);
+            var recipeName = "IsdaCdsRecipe";
+            var recipe = new ConfigurationRecipe(
+                testScope,
+                recipeName,
+                mktContext,
+                pricingContext,
+                description: "Isda cds valuation demo"
+            );
+
+            // UPSERT the configuration recipe
+            var upsertRecipeRequest = new UpsertRecipeRequest(recipe, null);
+            var upsertRecipeResponse = _recipeApi.UpsertConfigurationRecipe(upsertRecipeRequest);
+            Assert.That(upsertRecipeResponse.Value, Is.Not.Null);
+
+            // CREATE the required market data, in this case a credit curve and a yield curve
+            string ycXml = File.ReadAllText("../../../tutorials/Ibor/ExampleMarketData/IsdaYieldCurve_USD_20200605.xml");
+            var ycOpaque = new OpaqueMarketData(ycXml, "xml", "Example isda yield curve", ComplexMarketData.MarketDataTypeEnum.OpaqueMarketData);
+            var upsertYcId = new ComplexMarketDataId("Lusid", effectiveAt: testNow, marketAsset: "IsdaYieldCurve/USD");
+            var ccData = new CreditSpreadCurveData(
+                baseDate: testNow,
+                domCcy: "USD",
+                tenors: new List<string> {"6M", "1Y", "5Y", "10Y"},
+                spreads: new List<decimal> {0.001m, 0.0011m, 0.0015m, 0.002m},
+                recoveryRate: 0.4m,
+                marketDataType: ComplexMarketData.MarketDataTypeEnum.CreditSpreadCurveData);
+            var upsertCcId = new ComplexMarketDataId("Lusid", effectiveAt: testNow, marketAsset: "IsdaCreditCurve/XYZCorp/USD");
+
+            // UPSERT market data
+            var upsertYcRequest = new UpsertComplexMarketDataRequest(upsertYcId, ycOpaque);
+            var upsertCcRequest = new UpsertComplexMarketDataRequest(upsertCcId, ccData);
+            var upsertCmdResponse = _complexMarketDataApi.UpsertComplexMarketData(testScope,
+                new Dictionary<string, UpsertComplexMarketDataRequest>
+                {
+                    {"yc", upsertYcRequest},
+                    {"cc", upsertCcRequest}
+                });
+            Assert.That(upsertCmdResponse.Failed, Is.Empty);
+
+            // CREATE a inline valuation request with an example CDS instrument
+            var pvKey = "Holding/default/PV";
```

Comment:
Given we're looking to retire this key, does Valuation/PV not work?

Reply:
The key doesn't get populated unless you also ask for Holding/default/PV, which defeats the point.

The reason is that IsdaCds can only do Holding/default/PV, and lusid probably masks Valuation/PV with that value at the end. If you only ask for Valuation/PV then IsdaCds doesn't know what to calculate.